src/csharp/Microsoft.Spark.E2ETest/UdfTests/BroadcastTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.UdfTests
+{
+    [Serializable]
+    public class BroadcastTestClass
```

Comment:
Can you rename this?  I find it confusing that there's `class BroadcastTestClass` and also `class BroadcastTests`.  Maybe it should be called something like `BroadcastVariableExampleType`?

Reply:
Makes sense. How about `BroadcastExampleType` ?